lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -199,6 +200,8 @@
 KEY_MANA_DRIVER_ENABLED = "mana_driver_enabled"
 KEY_NVME_ENABLED = "nvme_enabled"
 ATTRIBUTE_FEATURES = "features"
+KEY_VMM_VERSION = "vmm_version"
```

Comment:
Move the keys to platform_utils too, so it can be shared.

Reply:
@squirrelsc  done.